dwave/system/samplers/dwave_sampler.py

```diff
@@ -180,6 +182,19 @@ def __init__(self, failover=False, retry_interval=-1, **config):
         self.client = Client.from_config(**config)
         self.solver = self._get_solver(penalty=self._solver_penalty)
 
+    def close(self):
+        """Close the underlying cloud-client in order to release system
+        resources like threads.
+
+        .. note::
+
+            The method blocks for all the currently scheduled work (sampling
+            requests) to finish.
+
+        See: :meth:`~dwave.cloud.client.base.Client.close`.
```

Comment:
```suggestion
        See: :meth:`~dwave.cloud.client.Client.close`.
```

This one creates a hyperlink, the current one does not.

Also elsewhere below

Reply:
That's interesting, because `Client` is defined in `base` (although it's imported in `client` namespace as well).